docs/installation/cloud-ex-machine-ocean.md

```diff
@@ -0,0 +1,239 @@
+
+### Step 1. Create a Digital Ocean account and log in
+
+If you have not done so already, go to <a href="https://digitalocean.com" target="_blank">Digital Ocean</a>, create an account, and log in.
+
+### Step 2. Generate a personal access token
+
+To generate your access token:
+
+  1. Go to the Digital Ocean administrator console and click **API** in the header.
+
+    ![Click API in Digital Ocean console](images/ocean_click_api.png)
+
+  2. Click **Generate New Token** to get to the token generator.
+
+    ![Generate token](images/ocean_gen_token.png)
+
+  3. Give the token a clever name (e.g. "machine"), make sure the **Write (Optional)** checkbox is checked, and click **Generate Token**.
+
+    ![Name and generate token](images/ocean_token_create.png)
+
+  4. Grab (copy to clipboard) the generated big long hex string and store it somewhere safe. (This is the personal access token you'll use in the next step to create your cloud server.)
+
+    ![Copy and save personal access token](images/ocean_save_token.png)
+
+### Step 3. Start Docker Machine
+
+1. If you have not done so already, install Docker Machine on your local host.
+
+  * <a href="https://docs.docker.com/engine/installation/mac/" target="_blank"> How to install Docker Machine on Mac OS X</a>
+
+  * <a href="https://docs.docker.com/engine/installation/windows/" target="_blank">How to install Docker Machine on Windows</a>
+
+  * <a href="https://docs.docker.com/machine/install-machine/" target="_blank">Install Docker Machine directly</a> (e.g., on Linux)
+
+2. At a command terminal, start Docker Machine and verify that it's running normally.
+
+  * Use `docker-machine ls` to get a list of Docker Machines and their status.
+
+            $ docker-machine ls
+            NAME      ACTIVE   DRIVER       STATE     URL   SWARM
+            default   -        virtualbox   Stopped    
+
+  * If Machine is stopped, start it.
+
+            $ docker-machine start default
+            (default) OUT | Starting VM...
+            Started machines may have new IP addresses. You may need to re-run the `docker-machine env` command.
+
+  * Set environment variables.
+
+            $ docker-machine env default
+            export DOCKER_TLS_VERIFY="1"
+            export DOCKER_HOST="tcp://xxx.xxx.xx.xxx:xxxx"
+            export DOCKER_CERT_PATH="/Users/londoncalling/.docker/machine/machines/default"
+            export DOCKER_MACHINE_NAME="default"
+            # Run this command to configure your shell:
+            # eval "$(docker-machine env default)"
+
+            eval "$(docker-machine env default)"
+
+  * Re-run `docker-machine ls` to check that it's now running.
+
+            $ docker-machine ls
+            NAME      ACTIVE   DRIVER       STATE     URL                         SWARM
+            default   *        virtualbox   Running   tcp:////xxx.xxx.xx.xxx:xxxx  
+
+  * Run a Docker command to make sure that Docker Engine is also up-and-running.
```

Comment:
Then, I think this is the final step...I'm saying the numbering/bulletting is odd.